internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -242,12 +252,22 @@ func (c *ClusterManager) processProvisioningClusters() []error {
 	// process each local pending cluster and compare to the underlying ocm cluster
 	for i := range provisioningClusters {
 		provisioningCluster := provisioningClusters[i]
-		glog.V(10).Infof("provisioning cluster ClusterID = %s", provisioningCluster.ClusterID)
-		metrics.UpdateClusterStatusSinceCreatedMetric(provisioningCluster, api.ClusterProvisioning)
-		_, err := c.reconcileClusterStatus(&provisioningCluster)
-		if err != nil {
-			errs = append(errs, errors.Wrapf(err, "failed to reconcile cluster %s status", provisioningCluster.ClusterID))
-			continue
+		if provisioningCluster.ClusterType != api.Enterprise.String() {
+			glog.V(10).Infof("provisioning cluster ClusterID = %s", provisioningCluster.ClusterID)
+			metrics.UpdateClusterStatusSinceCreatedMetric(provisioningCluster, api.ClusterProvisioning)
+			_, err := c.reconcileClusterStatus(&provisioningCluster)
+			if err != nil {
+				errs = append(errs, errors.Wrapf(err, "failed to reconcile cluster %s status", provisioningCluster.ClusterID))
+				continue
```

Comment:
Same as for the previous comment: this `continue` is not skipping anything and can be removed